web/apps/labelstudio/src/pages/CreateProject/Config/Preview.jsx

```diff
@@ -53,12 +62,14 @@ export const Preview = ({ config, data, error, loading, project }) => {
   }, [config]);
 
   const initLabelStudio = useCallback(async (config, task) => {
-    if (!task.data) return;
+    // wait for dependencies to load, the promise is resolved only once
+    // and is started when the component is mounted for the first time
+    await loadDependencies();
 
-    await resolvingEditor;
+    if (lsf.current || !task.data) return;
```

Comment:
`!task.data` im not sure this will calculate as expected `!{}` is false but so is `!{id:1}` is there a better way to test for this? does it start undefined and become an object after?

Reply:
![image](https://github.com/user-attachments/assets/71d5c34a-be3a-4256-8a5f-4aa63875aeca)

Reply:
@yyassi-heartex we specifically check for missing data (e.g. not loaded yet), not empty one, which is actually a valid data.